editor/py/serve.py

```diff
@@ -90,6 +97,34 @@ def upload_dataset(file_name):
   return 'ok'
 
 
+@app.route('/dataset/<file_name>/new', methods=['POST'])
+def new_dataset(file_name):
+  """Creates a new dataset in the db/ directory."""
+  path = f'db/{flask.g.user}/{file_name}.pbtxt'
+  if os.path.isfile(path):
+    return 'file exists'
+  with open(path, 'wb') as upload:
+    upload.write(b'\n')
+  return 'ok'
+
+
+@app.route('/dataset/enumerate', methods=['POST'])
+def enumerate_dataset():
+  """Creates a new dataset in the db/ directory based on a template reaction
+  pbtxt and a spreadsheet."""
```

Comment:
I think a one-line pydoc followed by elaboration is conventional and makes readable help.

Reply:
Expanded the docstring